### ISSUE TITLE
update_data not found

### ISSUE THREAD

**Issue by laresbernardo** [ModelOriented/forester#17]
Hi! Testing with the titanic example on README, I'm getting the following error:
```
Error in update_data(m, data_test[, -which(names(data_test) == target)],  : 
  could not find function "update_data"
```

**Comment by kozaka93** [comment]
Hi, thank you!
`update_data` is a function from the `DALEX` package. Library `DALEX` is necessary to prevent the mentioned error.

**Comment by laresbernardo** [comment]
Yes, makes sense. But I think you the function wasn't explicitly imported using `@importFrom DALEX update_data` to work OK.

**Comment by kozaka93** [comment]
fix in #18
